src/components/SignUp/SignUpForm.tsx

```diff
@@ -26,7 +26,7 @@ const SignUpForm = (): React.JSX.Element => {
 
 	const signup = async (name: string, email: string, password: string) => {
 		try {
-			const response = await fetch(`${apiUrl}sign-up`, {
+			const response = await fetch(`${BE_URL}auth/sign-up`, {
```

Comment:
```suggestion
			const response = await fetch(`/api/auth/sign-up`, {
```
Fetch resolves to the current host, so it will Just Work™